app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -479,6 +480,7 @@ class HomeFragment : Fragment() {
     override fun onResume() {
         super.onResume()
         hideToolbar()
+        OnVisualCompletenessExecutionQueue.post(sessionControlView!!.view)
```

Comment:
This doesn't take into account what happens if we restore in another Activity without going through `MainActivity`: I wrote up a potential solution here https://github.com/mozilla-mobile/fenix/issues/8034#issuecomment-587966952

tbh, I'm not sure if we can actually test that use case on device: afaik, Don't Keep Activities doesn't kill the app process like the system would.